pages/auth/login.vue

```diff
@@ -68,6 +68,8 @@
             {{ $t('auth.login.label') }}
           </hc-button>
         </form>
+        <p class="subtitle is-6">{{ $t('quotes.1.quoteTxt') }}</p>
+        <p class="subtitle is-7">{{ $t('quotes.1.quoteAuthor') }}</p>
```

Comment:
@christianeiselt can we remove the `1`? Just `$t('quotes.quoteText')`? Or if you want to add more quotes: $t('quotes.faceOfTheWorld.quoteTxt')?

Reply:
Is the goal to have only this one quote? Then this will be good enough, of course.
I used the number because I thought the goal would be to display a random quote and I'd say it will be difficult to come up for a shortname for every quote... But the number is static and therefore still not good for manual inserts.